VideoLocker/res/layout/fragment_handout.xml

```diff
@@ -1,27 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@color/grey_act_background"
-    android:orientation="vertical" >
 
-    <org.edx.mobile.view.custom.EdxWebView
-        android:id="@+id/webview"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:background="@color/grey_act_background" />
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I added padding to the webview itself. When you try to scroll on a longer handouts list, the scrolling animation is on the inner webview (the scrollbar and the blue bubble thingy when you scroll too far). 

Is there a better way to have that animation fill the screen?